mllib/src/main/scala/org/apache/spark/mllib/fpm/FPGrowth.scala

```diff
@@ -87,6 +88,14 @@ class FPGrowth private (
   }
 
   /**
+   * Indicates whether to mine item-sets or item-sequences (default: false, mine item-sets).
+   */
+  def setMineSequences(value: Boolean): this.type = {
```

Comment:
`value` -> `mineSequences` to be consistent with other setters in `spark.mllib` package.